tests/cpp/test_matmul_scheduler.cpp

```diff
@@ -3550,9 +3558,14 @@ INSTANTIATE_TEST_SUITE_P(
             MmaMacro::Hopper_64_128_16,
             MmaMacro::Hopper_64_64_16,
             MmaMacro::Hopper_64_32_16,
-            MmaMacro::Hopper_64_16_16), // mma_macros
+            MmaMacro::Hopper_64_16_16,
+            MmaMacro::Blackwell_128_256_16,
+            MmaMacro::Blackwell_128_128_16,
+            MmaMacro::Blackwell_128_64_16,
+            MmaMacro::Blackwell_128_32_16,
+            MmaMacro::Blackwell_128_16_16), // mma_macros
         testing::Values(1) // SplitK Factor
         ),
```

Comment:
```suggestion
    hopperTestNameSwizzle);
```
**[Suggested by AI]** Verified that the build will pass with this change.